appium/tests/screenobjects/inbox.screen.ts

```diff
@@ -29,27 +29,28 @@ class InboxScreen extends BaseScreen {
   }
 
   clickOnEmailBySubject = async (subject: string) => {
-    const selector =  `~${subject}`;
+    const selector = `~${subject}`;
     if (await (await $(selector)).isDisplayed() !== true) {
       await TouchHelper.scrollDown();
     }
     await ElementHelper.waitAndClick(await $(selector), 500);
   }
 
   clickCreateEmail = async () => {
-    if (await (await this.createEmailButton).isDisplayed() !== true ) {
+    await browser.pause(2000); // todo: loading inbox. Fix this: wait until loader gone
+    if (await (await this.createEmailButton).isDisplayed() !== true) {
       await TouchHelper.scrollDown();
       await (await this.createEmailButton).waitForDisplayed();
     }
```

Comment:
This needs a more proper fix - in inbox we should wait until loader is gone before interacting with/looking for the elements.